widget/embedded/src/utils/quote.ts

```diff
@@ -326,3 +293,60 @@ export const getUniqueBlockchains = (steps: Step[]) => {
 
   return result;
 };
+
+export const sortRoutesBy = (
+  strategy: SortStrategy,
+  routes: MultiRouteResponse['results']
+): MultiRouteResponse['results'] => {
+  return routes.sort((route1, route2) => {
+    const sortPreferenceType = ROUTE_SORTING_STRATEGY[strategy];
+
+    const route1Score =
+      route1.scores?.find(
+        (score) => score.preferenceType === sortPreferenceType
+      )?.score ?? 0;
+    const route2Score =
+      route2.scores?.find(
+        (score) => score.preferenceType === sortPreferenceType
+      )?.score ?? 0;
+
+    if (sortPreferenceType === 'NET_OUTPUT' || sortPreferenceType === 'PRICE') {
+      return route2Score - route1Score;
+    }
+    return route1Score - route2Score;
+  });
+};
```

Comment:
Please update the sort logic. you can check Rango Dapp.

Reply:
fixed

Reply:
Please review it once more.
Please utilize the name 'quote' instead of 'route'.

Reply:
fixed